recordedit/recordEdit.app.js

```diff
@@ -233,6 +234,7 @@
                             $log.warn(response);
                             throw response;
                         }).catch(function readCatch(exception) {
+                            // TODO: catchAll here too or should this be logic specific to recordedit?
                             ErrorService.errorPopup(exception.message, exception.code, "home page");
```

Comment:
I don't think a catchAll is necessary here since we're just redirecting to home page no matter what.

That is, unless we want to log it for monitoring/informational purposes. If so, then a catchAll could be put here to log it.